web/src/components/netflow-traffic.tsx

```diff
@@ -1291,6 +1295,7 @@ export const NetflowTraffic: React.FC<{
           {slownessReason()}
         </Alert>
       )}
+      <GuidedTourPopover id="netobserv" ref={guidedTourRef} />
```

Comment:
The guided tour is available for any components as soon as you provide `guidedTourRef` as props